src/main/java/uk/gov/companieshouse/company/metrics/exception/ControllerExceptionHandler.java

```diff
@@ -30,14 +39,14 @@
      */
     @ExceptionHandler({Exception.class})
     public ResponseEntity<Object> handleException(Exception ex, WebRequest request) {
-        LOGGER.error(ex.getMessage(), ex);
+        logger.error(ex.getMessage(), ex);
```

Comment:
maybe we can probably remove this line, and ex param below new error statement?

Reply:
this is addressed